### PR TITLE
New design and a few improvements

### PR DESCRIPTION
**Pull request by EvenDeveloper**

### CODE REVIEW

**Comment by ConorTheDev** [comment]
Thanks for the PR, will review in ~1h

File: Tweak.xm [hunk 1]
```diff
@@ -22,6 +21,11 @@
     [panelViewController.nowPlayingHeaderView.routingButton removeFromSuperview];
 }
 
+-(void)viewWillDisappear:(BOOL)animated {
```
Comment (ConorTheDev) [inline Tweak.xm]: What is this needed for?

File: Tweak.xm [hunk 2]
```diff
@@ -64,7 +68,7 @@
 
     destNav.navigationBarHidden = YES;
 
-    // Show the lyrics view controller as a popover
+        // Show the lyrics view controller as a popover
```
Comment (ConorTheDev) [inline Tweak.xm]: Fix comment indent please

File: Tweak.xm (hunk 1 above)
Comment (EvenDeveloper) [reply Tweak.xm]: Animated dismissal when the lockscreen disappears

**Pull request merged**